generator/src/lib.rs

```diff
@@ -441,6 +441,22 @@ impl quote::ToTokens for ConstVal {
         }
     }
 }
+
+// Interleaves a number, for example 100000 => 100_000. Mostly used to make clippy happy
+fn interleave_number(symbol: char, count: usize, n: &str) -> String {
+    let number: String = n
+        .chars()
+        .rev()
+        .enumerate()
+        .fold(String::new(), |mut acc, (idx, next)| {
+            if idx != 0 && idx % count == 0 {
+                acc.push(symbol);
+            }
+            acc.push(next);
+            acc
+        });
+    number.chars().rev().collect()
+}
```

Comment:
Would it make sense to explicitly disable this lint for vk.rs instead, since none of vk.rs is meant for direct human consumption anyway?

Reply:
I actually really like formatting the number to make it easier to read. It makes it easier to grok when debugging.